registerConfig.js

```diff
@@ -1,2 +1,2 @@
 /* This file is here to support backward compatability with previous verions */
-module.exports = require('./lib/iocConfigRegistration');
+module.exports = require('./src/iocConfigRegistration');
```

Comment:
Why is this change needed? This is referenced externally and should not be referenced from src. Breaks backward compat.